pkg/common/config/config.go

```diff
@@ -65,6 +65,8 @@ const (
 
 	// Default network provider for OSD
 	DefaultNetworkProvider = "OpenShiftSDN"
+
+	NonOSDe2eSecrets = "nonOSDe2eSecrets"
```

Comment:
Do you mind dropping a comment to explain the purpose of this constant?

Reply:
Especially since this viper key isn't meant to be used by our users, I think it's important to document that.

Reply:
ACK -- good point, done. :)